@AH-64D Apache Longbow/Addons/fza_ah64_weapons/functions/pylon/fn_SetPylonOwner.sqf

```diff
@@ -0,0 +1,24 @@
+/* ----------------------------------------------------------------------------
+Function: fza_weapons_fnc_SetPylonOwner
+
+Description:
+	
+Parameters:
+	
+Returns:
+	...
+
+Examples:
+	...
+
+Author:
+	Rosd6(Dryden)
+---------------------------------------------------------------------------- */
+params ["_heli","_Wassing"];
```

Comment:
Needs to be lower case `_wassing`. Also who is this with respect to? Gunner? Pilot?

I know the answer from looking down but shouldn't have to